apps/condo/pages/contact/[id]/update.tsx

```diff
@@ -41,7 +42,7 @@ ContactUpdatePage.requiredAccess = ContactsReadAndManagePermissionRequired
 export default ContactUpdatePage
 
 export const getServerSideProps: GetServerSideProps = async (context) => {
```

Comment:
you can avoid casting ID to string by passing props + params generic...

Se `resident-app/pages/meters/[id]/index` for reference:
```
GetServerSideProps<MeterPageProps, MeterPageParams>
```